columnar/src/column_values/mod.rs

```diff
@@ -135,6 +141,27 @@ pub trait ColumnValues<T: PartialOrd = u64>: Send + Sync {
     }
 }
 
+/// Empty column of values.
+pub struct EmptyColumnValues;
+
+impl<T: PartialOrd + Default> ColumnValues<T> for EmptyColumnValues {
+    fn get_val(&self, _idx: u32) -> T {
+        panic!("Internal Error: Called get_val of empty column.")
+    }
+
+    fn min_value(&self) -> T {
+        T::default()
```

Comment:
I think this should be a panic, so we don't use this accidentally to compute min max

Reply:
I disagree :)
min_value is misnamed though.

Reply:
It will yield incorrect min/max when empty columns are mixed in the computation